### PR TITLE
add Kong API Gateway exporter into exporters list

### PR DESCRIPTION
**Pull request by bsingr**
https://github.com/Kong/kong-plugin-prometheus

### CODE REVIEW

**Comment by brian-brazil** [comment]
I think this might better belong in the Software exposing Prometheus metrics list given it's a plugin rather than an exporter.

Metrics should indicate their unit, what are the units for bandwidth and latency?

**Comment by bsingr** [comment]
Bandwith is bytes, latencies is milliseconds.

**Comment by bsingr** [comment]
Agree + moved to `Software exposing Prometheus metrics`

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ separate exporters are needed:
    * [FreeBSD Kernel](https://www.freebsd.org/cgi/man.cgi?query=prometheus_sysctl_exporter&apropos=0&sektion=8&manpath=FreeBSD+12-current&arch=default&format=html)
    * [Grafana](http://docs.grafana.org/administration/metrics/)
    * [JavaMelody](https://github.com/javamelody/javamelody/wiki/UserGuideAdvanced#exposing-metrics-to-prometheus)
+   * [Kong API Gateway exporter](https://github.com/Kong/kong-plugin-prometheus)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: The link text should just be Kong

**Comment by brian-brazil** [comment]
Seconds are the standard unit for time in Prometheus, so those metrics could be improved.

**Comment by bsingr** [comment]
> Seconds are the standard unit for time in Prometheus, so those metrics could be improved.

Best practice is to add suffix `_milliseconds` or so?

**Comment by brian-brazil** [comment]
Best practice would be to divide it by 1000, and append `_seconds`.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**